tests/membership.py

```diff
@@ -184,7 +184,9 @@ def recovery_shares_scenario(args):
             test_remove_member(network, args, recovery_member=True)
             assert False, "Removing a recovery member should not be possible"
         except infra.proposal.ProposalNotAccepted as e:
-            assert e.proposal.state == infra.proposal.ProposalState.FAILED
+            # This is an apply() time failure, so the proposal remains Open
+            # since the last vote is effectively rolled back
```

Comment:
"rolled back" is confusing to me here (it implies that the consensus rolled it back?). Would "discarded" be clearer?

Reply:
Rolled back in the transaction/SQL sense, but we can call it discarded if you prefer?